.env.alfajoresstaging

```diff
@@ -13,11 +13,8 @@ CLUSTER_DOMAIN_NAME="celo-testnet"
 TESTNET_PROJECT_NAME="celo-testnet"
 
 BLOCKSCOUT_DOCKER_IMAGE_REPOSITORY="gcr.io/celo-testnet/blockscout"
-BLOCKSCOUT_WEB_DOCKER_IMAGE_TAG="web-f6c3e0888d1d0ef72dc8bf870808702b7fd13730"
-BLOCKSCOUT_INDEXER_DOCKER_IMAGE_TAG="indexer-f6c3e0888d1d0ef72dc8bf870808702b7fd13730"
+BLOCKSCOUT_DOCKER_IMAGE_TAG="9fcacb14cc0d68cb62478454ddb5c4ff9d66e92b"
 BLOCKSCOUT_WEB_REPLICAS=3
-# Increment this value everytime you redeploy blockscout. Or else the deployment will fail due to the
```

Comment:
why remove this warning here?

Reply:
Oops, didn't mean to do that. Just put it back